samples/runner_vs_vstest/measure.ps1

```diff
@@ -0,0 +1,26 @@
+$projects = Get-ChildItem $PSScriptRoot/*/*.csproj
+
+$projects | Foreach-Object { dotnet build $_ ; if ($LASTEXITCODE -ne 0) { throw "Build failed."} }
+
+$user = git config user.name
+$table = ""
+foreach ($project in $projects) { 
+    $dir = $project.Directory
+    $name = $project.BaseName 
+    $exe = $name + ".exe"
+    
+    $sw = [Diagnostics.Stopwatch]::StartNew()
+    & "$PSScriptRoot\..\..\artifacts\bin\$name\Debug\net8.0\$exe" --report-trx # --coverage
+    $runner = $sw.ElapsedMilliseconds; 
+    
+    $sw.Restart()
+    dotnet test $dir --no-restore --no-build --logger trx # --collect "Code Coverage"
+    $vstest = $sw.ElapsedMilliseconds
+
+    $entry = "|$name|$user|$vstest|$runner|$([math]::round($vstest/$runner * 100 , 0))%|`n"
```

Comment:
Nitpick (yeah I know :))

```suggestion
    $entry = "| $name | $user | $vstest | $runner | $([math]::round($vstest/$runner * 100 , 0))% |`n"
```

I'll update the other table to match this format.